tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,767 @@
+# familiar with MATLAB or would like to have Matplotlib automatically create
+# and manage parts of the visualization benefit from using the ``pyplot``
+# module to graph data. New users to Matplotlib may experience difficulty
+# understanding how elements of a visualization work together when using the
+# implicit approach.
+#
+# Examples
+# --------
+#
+# +------------------------------------+------------------------------------+
+# | Explicit, Object Oriented          | Implicit, ``pyplot``               |
+# | Programming (OOP)                  |                                    |
+# +====================================+====================================+
+# | ::                                 | ::                                 |
+# |                                    |                                    |
+# |     fig, ax = plt.subplots()       |    plt.plot([1,2,3],[1,2,3])       |
+# |     ax.plot([1,2,3],[1,2,3])       |                                    |
+# |                                    |                                    |
+# +------------------------------------+------------------------------------+
+# | `.pyplot.subplots` generates a     | :mod:`matplotlib.pyplot` creates   |
+# | `~.figure.Figure` and one or       | implicit Figure and Axes if        |
+# | more `~.axes.Axes` explicitly.     | there are no pre-existing          |
+# | `.Axes.plot` plots the data.       | elements and `.pyplot.plot` plots  |
+# |                                    | the data.                          |
+# +------------------------------------+------------------------------------+
+#
+#
+# Requirements
+# ============
+#
+# Matplotlib is a Python library and an installed version of *Python 3.6 or
+# higher* is required. Depending on your operating system, Python may already
+# be installed on your machine.
+#
+# Installing Maptlotlib is required in order to generate graphs with the
+# library. Install Matplotlib for your own development environment manually or
+# use a third-party package distribution.
+#
+# Third-party package distributions, such as
+# `Anaconda <https://www.anaconda.com/>`_,
+# `ActiveState <https://www.activestate.com/activepython/downloads>`_,
+# or `WinPython <https://winpython.github.io/>`_,
+# already provide Matplotlib and its dependencies. They have the added benefit
+# of including other scientific Python libraries as well. These packages work
+# as is and do not require additional installations.
+#
+# Installation from source
+# ------------------------
+#
+# In order to install Matplotlib from the source directory, run the
+# following command line executions using Python and installer program ``pip``
+# for the latest version of Matplotlib and its dependencies. This compiles
+# the library from the current directory on your machine. Depending on your
+# operating system, you may need additional support.
+#
+# ``python -m pip install matplotlib``
```

Comment:
We have like 4 install docs, so I'd rather link to https://matplotlib.org/devdocs/users/installing.html rather than have a fifth (even if it's totally stripped down):

Reply:
No problem! I can link to the Installation Guide at that block of text instead.